src/frontend/App.js

```diff
@@ -12,9 +13,16 @@ import PostingsRouter from './pages/postings/Postings.router';
 import SponsorsRouter from './pages/sponsors/Sponsors.router';
 import GeeseRouter from './pages/geese/Geese.router';
 import TeamDescriptionsRouter from './pages/team-descriptions/TeamDescriptions.router';
+import { addAuthTokenToRequests } from './api/server';
 
 const App = () => {
-  const token = useSelector(userSelectors.token);
+  let token = useSelector(userSelectors.token);
```

Comment:
This logic looks like it works fine, but you could shorten it to two lines with || and && operators:

```
const token = useSelector(userSelectors.token) || Cookies.get('tokenId');
token && addAuthTokenToRequests(token);

```